### ISSUE TITLE
Support long form for `image ls` command

### ISSUE THREAD

**Issue by asvetlov** [neuromation/platform-client-python#1334]
Print image URL in form available for direct usage by docker.

`neuro image ls -l` may produce a table, e.g.
```
image:myimage https://registry.neuro-ai-public.org.neu.ro/owner/myimage
```

**Comment by asvetlov** [comment]
Preferable to use formatter as we do for jobs and storage subsystems.

**Comment by asvetlov** [comment]
Fixed by #1354
